pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -262,6 +270,15 @@ stdenv.mkDerivation (fBuildAttrs // {
         targets = fBuildAttrs.bazelTargets;
       }
     }
+    ${
+      bazelCmd {
+        cmd = "run";
+        additionalFlags = fBuildAttrs.bazelRunFlags ++ ["--jobs" "$NIX_BUILD_CORES"];
```

Comment:
```suggestion
        additionalFlags = fBuildAttrs.bazelRunFlags ++ [ "--jobs" "$NIX_BUILD_CORES" ];
```

Reply:
I agree that the with-spaces formatting is preferable, but in this case I was intentionally following the formatting as is used in the rest of this file, esp in the lines immediately preceding this block

I'd be happy proceeding with either version

Reply:
done in https://github.com/NixOS/nixpkgs/pull/237648/commits/9114a2593e9f7a5f215a8692e7f73df903b0e133